src/components/data-processing/ConfigureEmbedding/ConfigureEmbedding.jsx

```diff
@@ -46,16 +48,25 @@ const ConfigureEmbedding = (props) => {
     [],
   );
 
+  const continuousEmbeddingPlots = ['mitochondrialContent', 'doubletScores'];
+
+  useEffect(() => {
```

Comment:
out of curiosity, why do we not need something like this for categorical plots?

Reply:
The data for categorical plots that exists right now are stored in `cellSets` and `embedding`, which are loaded by default. Data for continuous embedding plots are not loaded by default, hence we need to load them individually (line 54-59).

Reply:
makes sense, thanks for explaining!